modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/internal/model/listener/test/UserModelListenerTest.java

```diff
@@ -91,6 +98,34 @@ public void tearDown() throws Exception {
 		_companyLocalService.deleteCompany(_company.getCompanyId());
 	}
 
+	@Test
+	public void testOnAfterRemove() throws Exception {
+		UserTestUtil.addUser("default-service-account");
```

Comment:
@rijema , the `addUser` returns the user added, so you don't need to search the user again with `_userLocalService.getUserByScreenName`.